IntelliTect.TestTools.WindowsTestWrapper/IntelliTect.TestTools.WindowsTestWrapper/Utilities.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.IO;
+using System.Threading;
+
+namespace IntelliTect.TestTools.WindowsTestWrapper
+{
+    public class Utilities
+    {
+        /// <summary>
+        /// Parse a text file and return a string of its contents
+        /// </summary>
+        /// <param name="path">The full path of the location and file name</param>
+        /// <returns>Returns the text of the chosen file</returns>
+        public string ParseTextFile(string path)
+        {
+            string text = null;
+            int seconds = 0;
+            while (seconds < 10)
+            {
+                try
+                {
+                    text = File.ReadAllText(path);
+                }
+                catch (Exception e)
+                {
+                    Console.WriteLine("The file could not be read");
+                    Console.Write(e.Message);
+                }
+
+                if (text != null)
+                {
+                    break;
+                }
+
+                Thread.Sleep(500);
+                seconds++;
+            }
+            return text;
+        }
+
+        /// <summary>
+        /// Deletes a file given a location
+        /// </summary>
+        /// <param name="location">The location of the file without a slash at the end</param>
+        /// <param name="filename">The name and extension of the file</param>
+        public void DeleteDocument(string location, string filename)
+        {
+            try
+            {
+                File.Delete(location + "\\" + filename);
```

Comment:
Use Path.Combine here and you won't need to worry about if the `location` ends with a slash or not.